### PR TITLE
statistics: add some test cases of global-stats to cover more column types

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary: statistics: add some test cases of global-stats to cover more column types

### What is changed and how it works?

statistics: add some test cases of global-stats to cover more column types

### Check List 

Tests 

- Unit test

### Release note 

- statistics: add some test cases of global-stats to cover more column types

### CODE REVIEW

**Review by rebelice — COMMENTED** [review]
rest LGTM

File: statistics/histogram.go [hunk 1]
```diff
@@ -1831,7 +1831,11 @@ func MergePartitionHist2GlobalHist(sc *stmtctx.StatementContext, hists []*Histog
 			d.SetBytes(meta.Encoded)
 		} else {
 			var err error
-			_, d, err = codec.DecodeOne(meta.Encoded)
+			if types.IsTypeTime(hists[0].Tp.Tp) {
```
[the remark this answers is not in the document]
Comment (qw4990) [reply statistics/histogram.go:1834]: Some comments have been added, PTAL @rebelice .

**Review by Reminiscent — APPROVED** [review]
/LGTM

**Review by rebelice — APPROVED** [review]

**Pull request merged**